windows/wsl_distro_install_uninstall/install_wsl_distro.yml

```diff
@@ -0,0 +1,37 @@
+# Copyright 2023-2024 VMware, Inc.
+# SPDX-License-Identifier: BSD-2-Clause
+---
+# Description:
+# Install WSL distribution
+
+- name: "Set the command for installing WSL distribution"
+  ansible.builtin.set_fact:
+    install_distro_cmd: "wsl --install --name {{ wsl_distribution_name }} | Out-File -FilePath {{ wsl_file_path_win }}\\wsl_distro_install.txt"
```

Comment:
Doesn't here need to set the command according to `guest_os_product_type` or `guest_os_build_num`?

Reply:
The command is same for windows server and client and not related with the build number.